docs/previews.md

```diff
@@ -218,25 +330,29 @@ defined below:
 
 The `overrides` object can be passed any of the following keys. Please keep in
-mind that specifying your `linkResolver` and `htmlSerializer` is **required**.
-
-|      Key       | Required? |       Type        | Description                                                                                                                                                                                                                                                                 |
-| :------------: | :-------: | :---------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|  linkResolver  |    ✅     |     Function      | Determines how links in your preview content are resolved to URLs. If `pathResolver` is not defined, `linkResolver` is used to determine the `path` returned by `usePrismicPreview`. Generally, this should be the same function as the one provided in `gatsby-config`.    |
-| htmlSerializer |    ✅     |     Function      | Determines how rich text fields are resolved to HTML. Generally, this should be the same function as the one provided in `gatsby-config`.                                                                                                                                   |
-|  pathResolver  |           |     Function      | Function that allows for custom preview `path` resolving logic. This is useful if your `linkResolver` logic is different than how you generate URLs or paths for your pages. `pathResolver` receives the previewed document's raw API response from Prismic as an argument. |
-|   fetchLinks   |           | String or Array[] | Determines which link fields are fetched for the previewed document. For more information, refer to Prismic's docs on fetchLinks: https://prismic.io/docs/javascript/query-the-api/fetch-linked-document-fields                                                             |
-| repositoryName |           |      String       | Your Prismic repository name. Only provide this if you need to use a different repository name than the one you provided in `gatsby-config`.                                                                                                                                |
-|  accessToken   |           |      String       | Your Prismic access token. Only provide this if you need to use a different token than the one provided in `gatsby-config`.                                                                                                                                                 |
+mind that specifying your `linkResolver`, `htmlSerializer` and `repositoryName`
```

Comment:
Same as above, possible to make `htmlSerializer` optional?

Reply:
This was made optional in addition to `linkResolver` for previews. Previews only require `repositoryName` now 🎉